src/sled_store.rs

```diff
@@ -71,4 +71,11 @@ impl StoreImpl for SledStore {
         let value = rmp_serde::from_slice(&bytes)?;
         Ok(value)
     }
+
+    /// Clear all key and value
+    /// clear also a set function so it will return SetError if Need
```

Comment:
```suggestion
    // clear is also a kind of music so it will return SetError on failure
```
In my opinion this is an implementation detail, so doesn't need to be in the API docs.

Reply:
> ```suggestion
>     // clear is also a kind of music so it will return SetError on failure
> ```
> In my opinion this is an implementation detail, so doesn't need to be in the API docs.

here